app.js

```diff
@@ -0,0 +1,19 @@
+console.log('hello');
+
+
+//make new img 
+const newImg = document.createElement('img');
+//change it
+
+//newImg.setAttribute('src','https://www.placecage.com/300/300');
+
+newImg.setAttribute('src', 'https://source.unsplash.com/1600x900');
```

Comment:
you can make the the bonus by getting the height and width as a variable and using them as a part of the URL

```

let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```